Jenkinsfile.rdgo

```diff
@@ -10,18 +10,17 @@ def rdgo = "/srv/rhcos/output/rdgo"
 node(NODE) {
     docker.image(DOCKER_IMG).pull()
 
-    checkout scm
```

Comment:
We do this in the other jobs too...is it unnecessary there as well?

Reply:
In my test rdgo pipeline, Jenkins will checkout the `openshift/os` repo before starting any of the stages and then appears to mount it into the container.

Looking at the production jobs, it looks like the same thing is happening, so we should be able to make that change everywhere.

Example output from my test job:

```
Started by user Micah Abbott
Checking out git https://github.com/miabbott/os into /var/lib/jenkins/jobs/micah-coreos-rhcos-rdgo/workspace@script to read Jenkinsfile.rdgo
Wiping out workspace first.
Cloning the remote Git repository
Cloning repository https://github.com/miabbott/os
 > git init /var/lib/jenkins/jobs/micah-coreos-rhcos-rdgo/workspace@script # timeout=10
Fetching upstream changes from https://github.com/miabbott/os
 > git --version # timeout=10
 > git fetch --tags --progress https://github.com/miabbott/os +refs/heads/*:refs/remotes/origin/*
 > git config remote.origin.url https://github.com/miabbott/os # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/miabbott/os # timeout=10
Fetching upstream changes from https://github.com/miabbott/os
 > git fetch --tags --progress https://github.com/miabbott/os +refs/heads/*:refs/remotes/origin/*
 > git rev-parse refs/remotes/origin/dry_run_fork^{commit} # timeout=10
 > git rev-parse refs/remotes/origin/origin/dry_run_fork^{commit} # timeout=10
Checking out Revision d950ac9a99d7f6499a9dfeda8b3b631b38f10125 (refs/remotes/origin/dry_run_fork)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f d950ac9a99d7f6499a9dfeda8b3b631b38f10125
Commit message: "rdgo: archive all of the rdgo logs"
 > git rev-list --no-walk e9e4a39420c81f04b79a1708e604f063898a3280 # timeout=10
Running in Durability level: MAX_SURVIVABILITY
[Pipeline] node
Running on atomic-jslave-autobrew in /home/jenkins/workspace/micah-coreos-rhcos-rdgo
[Pipeline] {
[Pipeline] sh
[micah-coreos-rhcos-rdgo] Running shell script
+ docker pull quay.io/cgwalters/coreos-assembler
Using default tag: latest
Trying to pull repository quay.io/cgwalters/coreos-assembler ... 
sha256:6b2327b6f568cf2c83391240c14de86db966a4eaf5a05f9d9057d121a5036f61: Pulling from quay.io/cgwalters/coreos-assembler
bd02462c6d09: Already exists
e0955e3dd263: Already exists
4ac12fbd8a3b: Already exists
1f33f0024c51: Already exists
20c54c3dc16b: Already exists
Digest: sha256:6b2327b6f568cf2c83391240c14de86db966a4eaf5a05f9d9057d121a5036f61
Status: Image is up to date for quay.io/cgwalters/coreos-assembler:latest
[Pipeline] load
[Pipeline] { (pipeline-utils.groovy)
[Pipeline] }
[Pipeline] // load
[Pipeline] properties
[Pipeline] sh
[micah-coreos-rhcos-rdgo] Running shell script
+ docker inspect -f . quay.io/cgwalters/coreos-assembler
.
[Pipeline] withDockerContainer
atomic-jslave-autobrew seems to be running inside container f531de1a61d87b34f432371fa73208519a5e716ec3ef6801f3c058f9c9121c61
$ docker run -t -d -u 0:0 --net=host -v /srv:/srv --privileged -w /home/jenkins/workspace/micah-coreos-rhcos-rdgo --volumes-from f531de1a61d87b34f432371fa73208519a5e716ec3ef6801f3c058f9c9121c61 -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** -e ******** quay.io/cgwalters/coreos-assembler cat
$ docker top 3cc4215cd7b11c5374e894b5f899c43d5c16ec38b3de0f4af1fb8f99e858da5e -eo pid,comm
[Pipeline] {
[Pipeline] stage
[Pipeline] { (Provision)
[Pipeline] sh
[micah-coreos-rhcos-rdgo] Running shell script
...
```